tests/data/test_csm_data.py

```diff
@@ -51,6 +55,24 @@ def test_CsmData_cuda(csm_method, ellipse_phantom, random_kheader) -> None:
     assert relative_image_difference(torch.abs(csm.data), torch.abs(csm_ref.cuda())) <= 0.01
 
 
+@pytest.mark.parametrize('csm_method', [CsmData.from_idata_walsh, CsmData.from_idata_inati])
+def test_CsmData_downsampling(
+    csm_method: Callable, ellipse_phantom: EllipsePhantomTestData, random_kheader: KHeader
+) -> None:
+    """CsmData downsampling does not change smooth coil sensitivity maps."""
+    idata, csm_ref = multi_coil_image(n_coils=4, ph_ellipse=ellipse_phantom, random_kheader=random_kheader)
+
+    # Estimate coil sensitivity maps on original image size
+    smoothing_width = SpatialDimension(z=1, y=3, x=3)
+    csm = csm_method(idata, smoothing_width)
+
+    # Estimate coil sensitivity maps on low resolution image
```

Comment:
```suggestion
```
The comments just directly describe what the next line does anyways, right?

Instead, maybe comment why smoothing_width set to 1?
And name csm to csm_from_original `?